jump/src/context.rs

```diff
@@ -154,14 +155,32 @@ impl<'a> Context<'a> {
     fn prepare(&mut self, cmd: &'a Cmd) -> Result<(Process, Vec<Binding>, Vec<FileEntry>), String> {
         let process = self.prepare_process(cmd)?;
 
+        let mut load_entries = vec![];
+        for file in &self.lift.files {
+            if let Source::LoadBinding(binding_name) = &file.source {
+                let path = self.get_path(file);
+                load_entries.push(FileEntry::LoadAndInstall((
+                    self.prepare_process(
+                        self.lift
+                            .boot
+                            .bindings
+                            .get(binding_name)
+                            .ok_or_else(|| format!("No boot binding named {binding_name}."))?,
+                    )?,
+                    file.clone(),
+                    path,
+                )))
+            }
+        }
+
         let mut scie_tote = vec![];
         let mut file_entries = vec![];
         for (index, file) in self.lift.files.iter().enumerate() {
             if self.replacements.contains(&file) {
                 let path = self.get_path(file);
                 if file.size == 0 {
                     scie_tote.push((file.clone(), path));
-                } else {
+                } else if Source::Scie == file.source {
```

Comment:
Just confirming here that there is no longer a valid `else` if the file size is a non-zero, non-source

Reply:
That's right. The remaining case is `Source::LoadBinding(binding_name)` and all those were handled just above in the new `load_entries` population loop.